tests/TextTaggerTest.php

```diff
@@ -0,0 +1,193 @@
+<?php
+namespace Whatafix\Test;
+use PHPUnit\Framework\TestCase;
+use Whatafix\TextTagger\TextTagger;
+use Whatafix\TextTagger\TextTaggerV1;
+
+class TextTaggerTest extends TestCase
+{
+    private $textTagger;
+
+    public function __construct(?string $name = null, array $data = [], $dataName = '')
+    {
+        parent::__construct($name, $data, $dataName);
+        $this->textTagger = new TextTagger();
+    }
+
+    public function testInstance(): void
+    {
+        $class = get_class($this->textTagger);
+        $this->assertEquals("Whatafix\TextTagger\TextTagger", $class);
+    }
+
+    public function testTextTaggerGetTagsBasic(): void
+    {
+        $str = "Cette après-midi je suis allé manger une glace avec mes parents au parc. 
+        Puis nous avons fait une grande ballade.";
+        $tags = $this->textTagger->getTags($str);
+        $this->assertTrue(empty(array_diff(["family", "walk"], $tags)));
+    }
+
+    public function testNoDuplicateTags(): void
+    {
+        $str = "fils fille parent neveu";
+        $tags = $this->textTagger->getTags($str);
+        $this->assertEquals(1,sizeof($tags));

```

Comment:
assertCount devrait faire l'affaire
```suggestion
        $this->assertCount(1, $tags);
```